src/components/LoginPage/LoginPage.js

```diff
@@ -99,7 +111,7 @@ export default function LoginPage() {
                 <div className="col-md-6 bg-blue ">
                     <div className="d-flex flex-column align-items-center h-md-100  p-5 justify-content-center">
                         <h1 id="side-action" className="h1 text-white">Guest</h1>
-                        <p id="side-description" className="text-white h4 text-center">Enter to view real-time sensor information!</p>
+                        <p id="side-description" className="text-white h4 text-center">Enter to manage sensors!</p>
```

Comment:
This is illogical..? You log in as a guest to view information only, you don't manage any sensors.

Reply:
fixed ^^